packages/preferences-api/src/browser/utils.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export function deepFreeze(obj: any) {
```

Comment:
please copy `deepFreeze` from https://github.com/Microsoft/vscode/blob/master/src/vs/base/common/objects.ts,

I will also prefer `objects.ts` file name, to generic `utils.ts`.